indico/modules/categories/controllers/management.py

```diff
@@ -105,12 +108,69 @@ def _process(self):
         defaults = FormDefaults(self.category,
                                 meeting_theme=self.category.default_event_themes['meeting'],
                                 lecture_theme=self.category.default_event_themes['lecture'])
+
+        # Generate fields for existing translations
+        CategorySettingsForm.TITLE_FIELDS = ['title']
+        title_languages = get_all_locales()
+        for locale, translation in self.category.title_translations.items():
+            language = title_languages[locale]
+            language_text = f'{language[0]} ({language[1]})' if language[2] else language[0]
+            field_name = _('Title in ') + language_text
+            field_description = _('Users with ') + language_text + _(' as their language see this title.')
```

Comment:
Use placeholders and formatting (`_('...').format(...)`) for all such cases - you cannot use string concatenation for i18n, since by doing so, you:

- assume it will have the same structure in all languages
- assume translators will realize that these strings belong together and form a single sentance
- assume surrounding whitespace in translatable strings is preserved, AND translators keep it

The correct way of translating the description string here is this:

` _('Users with {} as their language see this title.').format(language_text)`

Reply:
Thanks, I've changed it.